test/tools/ossfuzz/yulProto_diff_ossfuzz.cpp

```diff
@@ -67,11 +83,14 @@ DEFINE_PROTO_FUZZER(Program const& _input)
 		// Parse protobuf mutated YUL code
 		if (!stack.parseAndAnalyze("source", yul_source) || !stack.parserResult()->code ||
 			!stack.parserResult()->analysisInfo)
+		{
+			printErrors(std::cout, stack.errors());
 			return;
+		}
 	}
 	catch (Exception const&)
 	{
-		return;
+		throw;
```

Comment:
What's the point of catching, if you only rethrow :-)?

Reply:
Good catch! :wink: